tests/runtime/probe

```diff
@@ -86,6 +86,13 @@ RUN {{BPFTRACE}} -e 'uprobe:/bin/bash:10 { printf("arg0: %d\n", arg0); exit();}'
 EXPECT Could not resolve address: /bin/bash:0xa
 TIMEOUT 5
 
+NAME uprobe_library
+RUN {{BPFTRACE}} -e 'uprobe:libc:write { printf("size: %d\n", arg0); exit(); }'
+EXPECT size: [0-9]+
+TIMEOUT 5
+AFTER cat /etc/passwd > /dev/null
```

Comment:
Can you write a testprog for this instead? Don't want this test to flake out on us in the future if cat changes.